### ISSUE TITLE
Some applications expects doc.kml in exported KMZ file

### ISSUE THREAD

**Issue by cyber-toad** [organicmaps/organicmaps#7398]
Meanwhile, OpenTracks expects doc.kml in KMZ files and don't import it otherwise.

_Originally posted by @rtsisyk in https://github.com/organicmaps/organicmaps/issues/995#issuecomment-1937116336_
            
Even for a single track/set of bookmarks stored internally in one kml file we export kmz. For now it just contains kml file that is fine according to the spec, but some applications expects that file name is doc.kml.
We can 
1) Try to discuss it with application teams asking them to fix their code
or
2) If there is no benefit from having initial file name for single-file export rename it to doc.kml as a work-around on our side.

**Comment by biodranik** [comment]
Did you test/confirm that the issue exists even with one kml inside a zip? I assume that @rtsisyk tested the import of all kml files in one zip, without an index file. @rtsisyk right?

If it's not the case, then the right solution is (1).

**Comment by cyber-toad** [comment]
> Did you test/confirm that the issue exists even with one kml inside a zip? I assume that @rtsisyk tested the import of all kml files in one zip, without an index file. @rtsisyk right?
> 
> If it's not the case, then the right solution is (1).

Yes, I selected single track in current Google Play OM build, shared it to OpenTracks and got an error about doc.kml.
